sympy/physics/control/control_plots.py

```diff
@@ -0,0 +1,48 @@
+from sympy.core.symbol import Symbol
+from sympy.external import import_module
+from sympy.integrals.transforms import inverse_laplace_transform
+from sympy.plotting import plot
+
+matplotlib = import_module(
+        'matplotlib', import_kwargs={'fromlist': ['pyplot']},
+        catch=(RuntimeError,))
+
+if matplotlib:
+    plt = matplotlib.pyplot
+
+def pole_zero(transfer_function, **kwargs):
+    r"""
+    Returns the Pole-Zero plot (also known as PZ Plot or PZ Map) of a system.
+    """
+    poles = transfer_function.poles()
+    zeros = transfer_function.zeros()
+
+    pole_points = [pole.as_real_imag() for pole in poles]
+    zero_points = [zero.as_real_imag() for zero in zeros]
+
+    x_poles = list(map(lambda x: x[0], pole_points))
+    y_poles = list(map(lambda x: x[1], pole_points))
+
+    x_zeros = list(map(lambda x: x[0], zero_points))
+    y_zeros = list(map(lambda x: x[1], zero_points))
+
+    plt.plot(x_poles, y_poles, 'x', mfc='none', markersize=15)
+    plt.plot(x_zeros, y_zeros, 'o', markersize=10)
```

Comment:
```suggestion
    plt.plot(x_poles, y_poles, 'x', mfc='none', markersize=10)
    plt.plot(x_zeros, y_zeros, 'o', markersize=7)
```
A small markersize will help us point out the zeros more precisely, no?